packages/palette/src/elements/Menu/Menu.tsx

```diff
@@ -53,23 +56,32 @@ const MenuContainer = styled(Box)`
 
 interface MenuItemProps extends BoxProps {
   children: React.ReactNode
+  fontSize?: SansSize
   href?: string
-  color?: string // TODO:  Look into type conflict with styled-system
+  textColor?: string // TODO:  Look into type conflict with styled-system
   onClick?: (event: React.MouseEvent<HTMLElement>) => void
+  paddingX?: number
+  paddingY?: number
+  textWeight?: "medium" | "regular"
```

Comment:
Maybe we can pull the type info from here? https://github.com/artsy/palette/blob/d74d283ae5833e346fd942d7037972b72c355693/packages/palette/src/elements/Typography/Typography.tsx#L131